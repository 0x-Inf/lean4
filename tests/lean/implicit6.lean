Variable f {A : Type} : A -> A -> A
Infixl 65 + : f
Show true + false
Show 10 + 20
Show 10 + (- 20)
Set pp::notation false
Set pp::coercion true
Show true + false
Show 10 + 20
Show 10 + (- 20)
