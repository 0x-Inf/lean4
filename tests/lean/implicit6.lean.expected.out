  Set: pp::colors
  Set: pp::unicode
  Assumed: f
⊤ + ⊥
10 + 20
10 + - 20
  Set: lean::pp::notation
  Set: lean::pp::coercion
f ⊤ ⊥
Nat::add 10 20
Int::add (nat_to_int 10) (Int::neg (nat_to_int 20))
